full_storage_utilization_test.py

```diff
@@ -0,0 +1,197 @@
+import time
+from sdcm.tester import ClusterTester
+
+
+class FullStorageUtilizationTest(ClusterTester):
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.soft_limit = 75
+        self.hard_limit = 90
+        self.sleep_time_before_autoscale = 120
+        self.sleep_time_for_balanced_tablets = 1800
+        self.stress_cmd_throttled = self.params.get('stress_cmd')
+        self.add_node_cnt = self.params.get('add_node_cnt')
+        self.auto_scaling_action_type = self.params.get('auto_scaling_action_type')
+
+    def prepare_dataset_layout(self, dataset_size, row_size=10240):
+        n = dataset_size * 1024 * 1024 * 1024 // row_size
+        seq_end = n * 100
+
+        return f'cassandra-stress write cl=ONE n={n} -mode cql3 native -rate threads=10 -pop dist="uniform(1..{seq_end})" ' \
+               f'-col "size=FIXED({row_size}) n=FIXED(1)" -schema "replication(strategy=NetworkTopologyStrategy,replication_factor=3)"'
+
+    def setUp(self):
+        super().setUp()
+        self.start_time = time.time()
+
+    def start_throttle_write(self):
+        self.run_stress_thread(stress_cmd=self.stress_cmd_throttled)
+        self.log.info("Wait for 2 mins for the stress command to start")
+        time.sleep(self.sleep_time_before_autoscale)        
+        
+    def scale_out(self):
+        self.start_slow_write()
```

Comment:
do we care about the performance of this? If so, will the output of this be visible in results?

Reply:
we don't want to performance numbers from this. But we want to ensure this is a throttled write. So that it won't fill-up data during topology change operation. @paszkow  mentioned we need to run some disk-io during add-node operation.  With his experiments while performing 2k write/second he ran into enospc error.